src/esp/gfx/RenderCamera.cpp

```diff
@@ -77,18 +77,22 @@ uint32_t RenderCamera::draw(MagnumDrawableGroup& drawables,
     return drawables.size();
   }
 
-  std::vector<std::pair<std::reference_wrapper<Mn::SceneGraph::Drawable3D>,
-                        Mn::Matrix4>>
-      drawableTransforms = drawableTransformations(drawables);
+  // dt is short for drawable transformations
+  using dt =
+      std::vector<std::pair<std::reference_wrapper<Mn::SceneGraph::Drawable3D>,
+                            Mn::Matrix4>>;
+  dt drawableTransforms = drawableTransformations(drawables);
```

Comment:
@mosra: drawableTransformations will create the vector from scratch every time? If we pre-allocate memory for the vector, and keep it in the entire cycle of the simulator, do you think it can save us time (in malloc)?

Reply:
This is on my "get rid of STL containers in all places" TODO list :)

On the other hand, from my experiments when removing usage of STL containers elsewhere, `std::allocator` usually does some pool allocation, thus the allocation overhead is not as big in subsequent (de)allocations. And since this is done just once in a frame, not a big deal at all.